### PR TITLE
Adding a check to not show the "Page #" title when there is only one.

### PR DESCRIPTION
**Pull request by danbev**
Motivation:
Currently when showing a forms app that only has one page, and does not
have a custom pageName, a title for the view is shown as 'Page 1'. When
there is only one page then we should not show this page name at all.

Modifications:
Added checks to see if there is a pageName set but

Result:
All test pass:
```shell
$ cd src/appforms
$ grunt testServer
```
Open [StepsView Test](http://127.0.0.1:3001/?grep=Backbone%20-%20Step%20View) in a browser.

JIRA:
https://issues.jboss.org/browse/RHMAP-4762

### CODE REVIEW

**Comment by danbev** [comment]
@nialldonnellyfh Would you mind reviewing this PR?

File: src/appforms/src/backbone/040-view06Steps.js [hunk 1]
```diff
@@ -71,12 +71,14 @@ StepsView = Backbone.View.extend({
 
     self.$el.find('li:eq(' + displayIndex + ')').addClass('active');
 
-    if(pageName.length === 0){
-      pageName = "Page " + (displayIndex + 1);
+    var hasCustomPageName = pageName.length !== 0;
```
Comment (nialldonnellyfh) [inline src/appforms/src/backbone/040-view06Steps.js]: I think this change is only hiding the page title. Am I understanding it correctly.

I think the idea is to hide the big page number at the top of the form if there is only a single page.

Comment (danbev) [inline src/appforms/src/backbone/040-view06Steps.js]: Ah, indeed it is. Let me update this.

Comment (danbev) [inline src/appforms/src/backbone/040-view06Steps.js]: @nialldonnellyfh Would you mind taking another look. I've updated this PR with an additional commit.

**Comment by danbev** [comment]
Having some issues with my cluster at the moment but I'll update it as soon as I have it running again so that this can be verified there.

File: dist/appForms-backbone.js [hunk 2]
```diff
@@ -4430,12 +4431,14 @@ StepsView = Backbone.View.extend({
 
     self.$el.find('li:eq(' + displayIndex + ')').addClass('active');
 
-    if(pageName.length === 0){
-      pageName = "Page " + (displayIndex + 1);
+    var hasCustomPageName = pageName.length !== 0;
+    if (displayIndex !== 0 || hasCustomPageName) {
```
Comment (nialldonnellyfh) [inline dist/appForms-backbone.js]: If there is no page title specified for the form, I don't think it should display the default "Page 1" title if there is only a single page.

**Comment by nialldonnellyfh** [comment]
Reviewed, small comment above.

File: dist/appForms-backbone.js (hunk 2 above)
Comment (danbev) [inline dist/appForms-backbone.js]: Sorry about this, but I rebased this prematurely. I've tried to fix this issue [here](https://github.com/feedhenry/fh-js-sdk/pull/164/commits/46db12691b423676ba9348a294d6690674d0e0f3#diff-c6c680480ef790b1aa733d3fd485e464R76) and also added an additional [test](https://github.com/feedhenry/fh-js-sdk/pull/164/commits/46db12691b423676ba9348a294d6690674d0e0f3#diff-7e2845e23b8af8e0cecc634dcf3e8e20R21) to verify. Was this what you had in mind?

**Comment by danbev** [comment]
@nialldonnellyfh I've finally got my cluster up and running again and got the following [app](http://testing.danbev.skunkhenry.com/#projects/wh2o5gj53hg3hxggvqdkrbaw/apps/wh2o5gmsxn4cftavsz5cwhsm/details) deployed.
I created it using [danbev/appforms-project-client.git](https://github.com/danbev/appforms-project-client.git) and the `RHMAP-4762-appforms-page-numbers` branch:

**Comment by nialldonnellyfh** [comment]
@danbev Verified the app in your cluster. The changes look fine.

Feel free to merge this back to master.

There needs to be a fh-ngui and template updates as per https://github.com/fheng/help/blob/master/developer_guides/appforms_feature_development/appforms_development.md#generated-files

**Pull request merged**